data/transport missions.txt

```diff
@@ -1828,6 +1828,50 @@ mission "Courier 1"
 		dialog phrase "generic cargo delivery payment"
 
 
+phrase "generic human package offer"
+	word
+		`A local`
+		`A man`
+		`A resident of <source>`
+		`A stranger`
+		`A woman`
+		`One of the spaceport's dock workers`
+	word
+		` `
+	word
+		`appears from seemingly nowhere to ask,`
+		`approaches you and says,`
+		`flags you down and asks,`
+		`walks up to you and asks,`
+	word
+		` `
+	word
+		`"Captain,`
+		`"Excuse me, Captain.`
+		`"Hello, Captain.`
+		`"Hello.`
+		`"Sorry to interrupt, but`
```

Comment:
You could probably move the double quote at the beginning of each of these words into the previous word.